code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -39,6 +39,14 @@
 	tastes = list("candy corn" = 1)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/candy_corn/prison
+	name = "desiccated candy corn"
+	desc = "If this candy corn were any harder Security would confiscate it for being a potential shiv."
+	force = 1 // the description isn't lying
+	throwforce = 1 // if someone manages to bust out of jail with candy corn god bless them
```

Comment:
Drat, now i need to also get rid of the candy corn thats in perma before tossing someone in there!